Makefile

```diff
@@ -27,4 +27,33 @@ test.integration:
 	INTEGRATION=1 go test ./...
 
 test.integration.verbose:
-	INTEGRATION=1 go test ./... -v
\ No newline at end of file
+	INTEGRATION=1 go test ./... -v
+
+# .PHONY : is an idiomatic way to differentiate commands from files in GNU Make
+.PHONY:
```

Comment:
Hello @V0idMatr1x, 

Thanks for your time <3

**Add each phony target as a prerequisite of .PHONY immediately before the target declaration, rather than listing all the phony targets in a single place.** 
Reference: [makefile-style-guide](https://clarkgrubb.com/makefile-style-guide)

Reply:
Hi Mario,

I will make the necessary changes, and I appreciate the review! 

Thank you for your time as well!